package.json

```diff
@@ -268,6 +268,11 @@
                             "cider/piggieback": "0.5.2"
                         }
                     },
+                    "calva.clojureLspVersion": {
+                        "type": "string",
+                        "default": "2021.03.30-20.42.34",
+                        "markdownDescription": "The version of `clojure-lsp` to download and use."
```

Comment:
The window/vscode must be reloaded for this change to take effect. Do you think we should inform of that here? The `Developer: Reload Window` command could be used, but I'm not sure what we should say here exactly.

To take it a step further (probably very unnecessarily), we could make changing this setting cause a popup prompt that mentioned VS Code must be reloaded, and allow them to press a button to reload, or not reload, like when installing an extension that requires a reload. But, it may be best to keep it simple and just inform the user in the description.